zrml/prediction-markets/src/benchmarks.rs

```diff
@@ -48,35 +49,28 @@ use zrml_market_commons::MarketCommonsPalletApi;
 // amount of native currency
 fn create_market_common_parameters<T: Config>(
     permission: MarketCreation,
-) -> Result<
-    (
-        T::AccountId,
-        T::AccountId,
-        MarketPeriod<T::BlockNumber, MomentOf<T>>,
-        MultiHash,
-        MarketCreation,
-    ),
-    &'static str,
-> {
+) -> Result<(T::AccountId, T::AccountId, MultiHash, MarketCreation), &'static str> {
     let caller: T::AccountId = whitelisted_caller();
-    let _ = T::AssetManager::deposit(Asset::Ztg, &caller, (u128::MAX).saturated_into());
+    T::AssetManager::deposit(Asset::Ztg, &caller, (u128::MAX).saturated_into()).unwrap();
```

Comment:
Just a suggestion , instead of unwrap() can we use map_err() or something similar and return error like "AssetManager deposit failed" ?